QUEST_JOBSTEP.tsv

```diff
@@ -109,42 +109,42 @@ QUEST_JOBSTEP_20150317_000108	You will find Merogs at the Fallen Worry Forest.{n
 QUEST_JOBSTEP_20150317_000130	Sculptor Tesla
 QUEST_JOBSTEP_20150317_000131	A sculptor's talent is important, but so is having an eye for seeing important materials. {nl}I will help you learn my skills if you bring me the blue crystals I'm looking for. 
-QUEST_JOBSTEP_20150317_000132	That is the problem. {nl}The Yekub swallowed it as soon as it was found on the 2nd Floor of the Miner's Village. 
-QUEST_JOBSTEP_20150317_000133	Oh, this is it. {nl}I have a lot of time so I will teach you thoroughly.
+QUEST_JOBSTEP_20150317_000132	$That is the problem. {nl}The Yekub swallowed it as soon as it was found on the 2nd Floor of the Miners' Village. 
+QUEST_JOBSTEP_20150317_000133	$Oh, this is it. {nl}I have a lot of time, so I will teach you thoroughly.
 QUEST_JOBSTEP_20150317_000134	You want to learn the swordsmanship of a Highlander from me? {nl}I will consider it if you can get cooperation from the Peltastas.
 QUEST_JOBSTEP_20150317_000135	As compensation for seeking cooperation, just support 5 boards of wooden buckler type.{nl}Defeating Upents at Indigo Trees Plateau will be enough.
 QUEST_JOBSTEP_20150317_000136	Thank you for the supplies, but the Highlander Master wouldn't have sent you for nothing. {nl}Tell me what you want. {nl}
 QUEST_JOBSTEP_20150317_000137	An overall cooperation between the Masters. Alright. {nl}Tell him that the Peltastas are for it. 
 QUEST_JOBSTEP_20150317_000138	If the Masters unite, it won't be long until the kingdom rises again. {nl}You did a good job. I will teach you my swordsmanship.
 QUEST_JOBSTEP_20150317_000139	That is enough. {nl}Get the Peltasta's cooperation. 
-QUEST_JOBSTEP_20150317_000140	We Peltastas promised the Highlanders we would lend our force to rebuild the kingdom.{nl}So if you also want to learn from me, you should be also responsible.{nl}
+QUEST_JOBSTEP_20150317_000140	$We Peltastas promised the Highlanders we would lend our forces to rebuild the kingdom.{nl}So if you want to learn from me, you should also be responsible.{nl}
 QUEST_JOBSTEP_20150317_000141	Go meet Uska and lend your force.
 QUEST_JOBSTEP_20150317_000142	So you came here at the request of the Peltasta Master.{nl}The Highland Master once guaranteed that he would get support from the other masters. {nl}
 QUEST_JOBSTEP_20150317_000143	We, Peltastas, consider our duties as humans to be most important. 
 QUEST_JOBSTEP_20150317_000144	We, Peltastas, consider our duties as humans to be most important. 
 QUEST_JOBSTEP_20150317_000145	I was about to ask for aid in the search. {nl}
 QUEST_JOBSTEP_20150317_000146	I sent 4 scouts to the Spine Heart Dale, but they all disappeared.{nl}Maybe they didn't think the thorny forest was that dangerous... Please find them.
-QUEST_JOBSTEP_20150317_000147	Some issuse may have emerged.{nl}Please go to the Spine Heart Dale quickly.
+QUEST_JOBSTEP_20150317_000147	$Some issue may have emerged.{nl}Please go to the Spine Heart Dale quickly.
```

Comment:
They can emerge or arise; they're basically the same thing. In this case, it would be "Some issue may have arisen."